test/unit/get-commit-test.js

```diff
@@ -48,3 +48,24 @@ test('get commit request succeeds', t => {
     t.end()
   })
 })
+
+test('get commit fails', t => {
+  const state = {
+    api,
+    debug: () => {}
+  }
+  simple.mock(api.repos, 'getCommit').rejectWith({
+    message: 'Not Found'
```

Comment:
the error message is not `Not Found` it looks more like `'{"message":"Not Found","documentation_url":"https://developer.github.com/v3"'`. But you can set `status: 'Not Found'` and then test for `error.status` below, or set `code: 404` and check for `error.code` below, like in the other test